astrodb_utils/utils.py

```diff
@@ -68,21 +67,22 @@ class AstroDBError(Exception):
 #     yield
 #     sys.tracebacklimit = default_value  # revert changes
 
-# TODO:  Where should these live?
-REFERENCE_TABLES = [
-    "Publications",
-    "Telescopes",
-    "Instruments",
-    "Modes",
-    "PhotometryFilters",
-    "Versions",
-    "Parameters",
-]
-
 
-def load_astrodb(db_file, recreatedb=True, reference_tables=REFERENCE_TABLES):
+def load_astrodb(db_file, data_path="data/", recreatedb=True, reference_tables=None):
```

Comment:
As an alternative, we can have reference_tables have a default value instead of None. It's a bit of a cleaner approach, though misses out on providing a message if none was provided.